### PR TITLE
Timeout Docs Update

### PR DESCRIPTION
**Pull request by sarahalsmiller**
### Description
There was a mutation in how this value is parsed at some point between when I developed it and when it was released. The code isn't broken, but the docs are incorrect and will cause the user to receive an error if they follow the format listed here.

### PR Checklist

* [ ] updated test coverage
* [ ] external facing docs updated
* [ ] appropriate backport labels added
* [ ] not a security concern

### CODE REVIEW

File: website/content/docs/connect/gateways/api-gateway/configuration/routetimeoutfilter.mdx [hunk 1]
```diff
@@ -21,8 +21,8 @@ The following list outlines field hierarchy, data types, and requirements in a `
    - [`name`](#metadata-name): string | required 
    - [`namespace`](#metadata-namespace): string  | `default`
 - [`spec`](#spec): map | required
-  - [`idleTimeout`](#spec-idletimeout): number | `0`
-  - [`requestTimeout`](#spec-retryonconnectfailure): number | `0`
+  - [`idleTimeout`](#spec-idletimeout): time.Duration | `""`
+  - [`requestTimeout`](#spec-retryonconnectfailure): time.Duration | `""`
```
Comment (trujillo-adam) [inline website/content/docs/connect/gateways/api-gateway/configuration/routetimeoutfilter.mdx:25]:
```suggestion
  - [`idleTimeout`](#spec-idletimeout): string | `""`
  - [`requestTimeout`](#spec-retryonconnectfailure): string | `""`
```
I think we still need to label these as strings in the configuration model. My understanding is that any value placed in quotation marks is considered a string.

File: website/content/docs/connect/gateways/api-gateway/configuration/routetimeoutfilter.mdx [hunk 2]
```diff
@@ -97,18 +97,18 @@ Map that contains the details about the gateway policy. The `apiVersion`, `kind`
 
 ### `spec.idleTimeout
 
-Specifies the total amount of time permitted for the request stream to be idle.
+Specifies the total amount of time permitted for the request stream to be idle. Must specify a parseable number and a unit. Example: "5s"
```
Comment (trujillo-adam) [inline website/content/docs/connect/gateways/api-gateway/configuration/routetimeoutfilter.mdx:100]:
```suggestion
Specifies the total amount of time permitted for the request stream to be idle. Must specify a parseable number and a unit, for example "5s". 
```
If the default value is an empty string, what is the behavior? Does that mean there is no timeout and the stream can remain idle indefinitely?

File: website/content/docs/connect/gateways/api-gateway/configuration/routetimeoutfilter.mdx [hunk 3]
```diff
@@ -97,18 +97,18 @@ Map that contains the details about the gateway policy. The `apiVersion`, `kind`
...
 
-- Default: 0
-- Data type: Integer
+- Default: ""
+- Data type: string
 
 ### `spec.requestTimeout`
 
-Specifies the total amount of time in nanoseconds, including retry attempts, Consul permits for the entire downstream request to be processed.
+Specifies the total amount of time in nanoseconds, including retry attempts, Consul permits for the entire downstream request to be processed. Must specify a parseable number and a unit. Example: "5s"
```
Comment (trujillo-adam) [inline website/content/docs/connect/gateways/api-gateway/configuration/routetimeoutfilter.mdx:109]:
```suggestion
Specifies the total amount of time in nanoseconds, including retry attempts, Consul permits for the entire downstream request to be processed. Must specify a parseable number and a unit, for example "5s". 
```

**Review by trujillo-adam — COMMENTED** [review]
Just a few nits and a question about the default behavior.

File: website/content/docs/connect/gateways/api-gateway/configuration/routetimeoutfilter.mdx (hunk 1 above)
Comment (sarahalsmiller) [reply website/content/docs/connect/gateways/api-gateway/configuration/routetimeoutfilter.mdx:25]: You're correct, I meant to change that back.

File: website/content/docs/connect/gateways/api-gateway/configuration/routetimeoutfilter.mdx (hunk 2 above)
Comment (sarahalsmiller) [reply website/content/docs/connect/gateways/api-gateway/configuration/routetimeoutfilter.mdx:100]: I believe if its not set it defaults to whatever envoy defaults to when not specified, which according to the envoy docs appears to be 1 hour https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/protocol.proto#envoy-v3-api-field-config-core-v3-httpprotocoloptions-idle-timeout.

**Review by trujillo-adam — COMMENTED** [review]
Looks good

**Review by missylbytes — APPROVED** [review]

**Pull request merged**